src/chrome/content/rules/trvl-px.com.xml

```diff
@@ -0,0 +1,23 @@
+<!--
```

Comment:
We use to add a reference back to the 'parent' ruleset too. (Same for pending #13976 and #13977.)

Reply:
Added reference to `Expedia.com.xml` which I'll rename from `Expedia.xml` in separate PR. Current working on its ccTLD.